bitnami/magento/Chart.yaml

```diff
@@ -11,6 +11,8 @@ dependencies:
   - condition: mariadb.enabled
     name: mariadb
     repository: https://charts.bitnami.com/bitnami
+    tags:
+      - magento-database
     version: 9.x.x
   - condition: elasticsearch.enabled
     name: elasticsearch
```

Comment:
ES is set to 12.X.X, while the latest major branch is 13.X.X. Changes between 12 and 13 are related to the Helm 2 to Helm 3 migration, see https://github.com/bitnami/charts/tree/master/bitnami/elasticsearch#to-1300, but probably we forgot to bump the subchart version in the past. I think it is safe to use the new major now, there are no backwards compatibility issues due to those changes